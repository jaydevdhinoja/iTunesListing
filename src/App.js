import React from "react";
import Routes from "./config/routes";
import { BrowserRouter } from "react-router-dom";

import "./styles/main.sass";
import Header from "./components/Header";
import Footer from "./components/Footer";

import GlobalListingProvider from './ContextProvider'

const App = () => (
  <>
    <BrowserRouter>
      <GlobalListingProvider>
        <Header />
        <Routes />
        <Footer />
      </GlobalListingProvider>
    </BrowserRouter>
  </>
);

export default App;