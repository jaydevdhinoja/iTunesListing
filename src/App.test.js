import { renderWithContext, InitialState } from './tests/common';
import App from './App';

import { cleanup } from '@testing-library/react';

afterEach(cleanup);

test('renders learn react link', () => {
  const { container } = renderWithContext(<App />, { value: InitialState });

		expect(container).toMatchSnapshot();
  
});
