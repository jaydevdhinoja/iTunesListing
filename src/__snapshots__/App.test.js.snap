// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="container d-flex justify-content-between align-items-center"
    >
      <div
        class="d-flex justify-content-between"
      >
        <a
          href="/"
        >
          <span
            class="logo"
          >
            <h1>
              Top Albums
            </h1>
          </span>
        </a>
        <a
          href="/songs"
        >
          <span
            class="logo"
          >
            <h1>
              Top Songs
            </h1>
          </span>
        </a>
      </div>
      <a
        href="/favourites"
      >
        <li
          class="header-menu__item link link--white"
        >
          Favourites
        </li>
      </a>
    </div>
  </div>
  <section
    class="albums-grid container"
  >
    <div
      class="albums-grid__headline section-headline"
    >
      <h2>
        Top Albums
      </h2>
      <div
        class="albums-grid__headline-search"
      >
        <input
          class="search-input"
          placeholder="Search"
          type="text"
          value=""
        />
        <img
          alt="Search"
          src="search-solid.svg"
        />
      </div>
    </div>
    <ul
      class="albums-grid__list"
    />
  </section>
  <section
    class="footer"
  >
    <div
      class="footer__content container "
    >
      <p>
        Developed by Jaydev Dhinoja
      </p>
    </div>
  </section>
</div>
`;
